src/app/faq/components/faq-page/faq-page.component.spec.ts

```diff
@@ -0,0 +1,33 @@
+import {ComponentFixture, TestBed} from '@angular/core/testing';
+
+import {FaqPageComponent} from './faq-page.component';
+import {FaqBlockComponent} from '../faq-block/faq-block.component';
+import {FaqItemComponent} from '../faq-item/faq-item.component';
+import {BreadcrumbModule} from '@shared/breadcrumb/breadcrumb.module';
+import {RouterTestingModule} from '@angular/router/testing';
+
+describe('FaqPageComponent', () => {
+  let component: FaqPageComponent;
+  let fixture: ComponentFixture<FaqPageComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [FaqPageComponent, FaqBlockComponent, FaqItemComponent],
+      imports: [
+        BreadcrumbModule,
+        RouterTestingModule.withRoutes([]),
+      ],
```

Comment:
This test is currently failing because of a missing import - you want to add the `HttpClientModule` here, since the component is using `HttpClient`.

Reply:
Fixed